openssl-dynamic/src/main/java/org/apache/tomcat/jni/SSLContext.java

```diff
@@ -312,7 +312,21 @@ public static native boolean setCertificateBio(
     public static native long sessionTimeouts(long ctx);
 
     /**
-     * Set TLS session keys.
+     * TLS session ticket key resumption statistics.
+     */
+    public static native long sessionTicketKeyNew(long ctx);
+    public static native long sessionTicketKeyResume(long ctx);
+    public static native long sessionTicketKeyRenew(long ctx);
+    public static native long sessionTicketKeyFail(long ctx);
```

Comment:
no: the underlying data is `uint32` so `int` does not have enough space. Also, the other stats are all `long`